src/model/CatalyticActivity.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react';
+import UniProtEvidenceTag from '../components/UniProtEvidenceTag';
+import { EvidenceType } from './types/modelTypes';
+
+type CatalyticActivityData = {
+  comments: [
+    {
+      commentType: string;
+      reaction: {
+        commentType: string;
+        name: string;
+        reactionReferences: Array<{ databaseType: string; id: string }>;
+        ecNumber: string;
+        evidences: EvidenceType[];
+      };
+    }
+  ];
+};
+
+type CatalyticActivityProps = {
+  data: CatalyticActivityData;
+};
+
+export const CatalyticActivity: React.FC<CatalyticActivityProps> = ({
+  data,
+}) => {
+  const catalyticActivityData = data.comments.filter(
+    d => d.commentType === 'CATALYTIC ACTIVITY'
+  );
+  if (!catalyticActivityData || catalyticActivityData.length <= 0) {
+    return null;
+  }
+  return (
+    <Fragment>
+      <h4>Catalytic Activity</h4>
+      {catalyticActivityData.map(catalyticActivity => (
+        <p key={`catalytic-${catalyticActivity.reaction.ecNumber}`}>
+          <strong>{catalyticActivity.reaction.ecNumber}</strong>{' '}
+          {catalyticActivity.reaction.name}
+          {catalyticActivity.reaction.evidences.map(evidence => (
```

Comment:
This will be coming from the server - do we want to do any checking to make sure that all of the keys in this path exist?